exporter/metric/stdout/stdout.go

```diff
@@ -107,11 +107,13 @@ func NewRawExporter(config Config) (*Exporter, error) {
 
 // InstallNewPipeline instantiates a NewExportPipeline and registers it globally.
 // Typically called as:
-// pipeline, err := stdout.InstallNewPipeline(stdout.Config{...})
-// if err != nil {
-// 	...
-// }
-// defer pipeline.Stop()
+//
+// 	pipeline, err := stdout.InstallNewPipeline(stdout.Config{...})
+// 	if err != nil {
+// 		...
+// 	}
+// 	defer pipeline.Stop()
+//
 // ... Done
```

Comment:
In your other PRs you indented this line as well. Should this be indented?

Reply:
@MrAlias fixed